ci/linux_openresty_common_runner.sh

```diff
@@ -101,6 +101,10 @@ after_success() {
     echo "done"
 }
 
+linux_get_dependencies() {
+    install_dependencies
```

Comment:
We can put this in each runner.sh's `do_install`? So there is no need to put `sudo ./ci/${{ matrix.os_name }}_runner.sh linux_get_dependencies` in each workflow.